buttoncount.js

```diff
@@ -1,4 +1,11 @@
-function(){
+function fun(){
```

Comment:
This function is not indentated properly. 
Why so much extra empty lines. 
Don't direct copy,paste from internet, atleast after getting code from internet, type it yourself